contracts/arena/DFArenaGetterFacet.sol

```diff
@@ -98,13 +116,20 @@ contract DFArenaGetterFacet is WithStorage, WithArenaStorage {
         return arenaStorage().gameover;
     }
 
+    function getStartTime() public view returns (uint256) {
+        return arenaStorage().startTime;
+    }
+
     function getEndTime() public view returns (uint256) {
         return arenaStorage().endTime;
     }
 
     function getRoundDuration() public view returns (uint256) {
+        if(arenaStorage().startTime == 0) {
```

Comment:
What if we make this a require instead `require(startTime != 0, "round has not started yet")` for consistency?

Reply:
Why? if the round hasnt started the round duration is 0. there is no need to throw an error.